_data/navigation.yml

```diff
@@ -29,8 +29,6 @@ setup:
     children:
       - title: Demo Install
```

Comment:
Do we still need children under this menu item if there's only one? I.e. would it be possible to just make the "Quickstarts" text take you straight to the "here's a list of things to try out" page?